public/lesswrong-editor/src/editorConfigs.js

```diff
@@ -154,6 +155,7 @@ export const postEditorConfig = {
 			'|',
 			'trackChanges',
 			'math',
+			'ctaButtonToolbarItem',
```

Comment:
I think we'll want to make sure these additions don't show up on LW if the plugin itself is forum-gated